src/main/resources/assets/minecolonies/lang/en_US.lang

```diff
@@ -179,3 +179,8 @@ com.minecolonies.gui.workerHuts.farmerHut.unused=unused
 com.minecolonies.gui.scarecrow.user=Field used by: %s
 com.minecolonies.gui.scarecrow.user.noone=no-one
 com.minecolonies.gui.buildtool.nohutininventory=The hut was not found in your inventory.
+
+com.minecolonies.gui.debugScreen.noCloseColony=No close colony!
+com.minecolonies.gui.debugScreen.nextColony=Next Colony in: %d.
+com.minecolonies.gui.debugScreen.blocksFromCenter=%d blocks from the center.
+com.minecolonies.gui.debugScreen.required=%d required to place a Colony.
```

Comment:
The text will be: Next colony in: %d (%d required to place a colony)
Blocks would be doubled